server/controllers/feedbackController.js

```diff
@@ -0,0 +1,26 @@
+const feedBackModel = require('../models/Feedback');
+
+function createFeedback(req, res) {
+  const feedback = feedBackModel.addFeedback(req);
+  feedback.then((data) => {
+    if (data.error) {
+      res.status(400).json({ error: data.error });
+    } else {
+      res.json({ feedback: data.feedback });
+    }
+  });
+}
+
+function getFeedback(req, res) {
+  const interviewId = req.params.interviewId;
+  const feedback = feedBackModel.getFeedback(interviewId);
+  feedback.then((data) => {
+    if (data.error) {
+      res.status(400).json({ error: data.error });
+    } else {
+      res.json({ interview: data.interview });
```

Comment:
I don't think we need getFeedback to return an object containing the interview if we're just going to do `data.interview`. I think it'd be cleaner to just return the interview itself from getFeedback and then here we can just use `data`